cmake/ycm-0.4.0/modules/AddInstallRPATHSupport.cmake

```diff
@@ -100,6 +99,19 @@ function(ADD_INSTALL_RPATH_SUPPORT)
                              "${_multiValueArgs}"
                              "${ARGN}")
 
+  # if either RPATH is disabled or INSTALL_RPATH is disable
+  # and the INSTALL_NAME_DIR variable is set, then hardcode the install name
+  if(CMAKE_SKIP_RPATH OR CMAKE_SKIP_INSTALL_RPATH)
+    if(DEFINED _ARS_INSTALL_NAME_DIR)
+      set(CMAKE_INSTALL_NAME_DIR ${_ARS_INSTALL_NAME_DIR} PARENT_SCOPE)
+    endif()
```

Comment:
I don't know. There is a possible case which is without RPATH and without absolute path, and I think is the one covered by the fact that we allow the variable to be undefined